bookwyrm/models/book.py

```diff
@@ -41,6 +41,9 @@ class BookDataModel(ObjectMixin, BookWyrmModel):
     openlibrary_key = fields.CharField(
         max_length=255, blank=True, null=True, deduplication_field=True
     )
+    finna_key = fields.CharField(
```

Comment:
Ideally this would be visible in `templates/book/book_identifiers.html` and perhaps also `templates/book/edit/edit_book_form.html`.